src/CoreFx.Private.TestUtilities/src/System/AdminHelpers.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using Xunit;
+
+namespace System
+{
+    public static class AdminHelpers
+    {
+        /// <summary>
+        /// Runs the given command as sudo
+        /// </summary>
+        /// <param name="commandLine">The command line to run as sudo</param>
+        /// <returns> Returns the process exit code (0 typically means it is successful)</returns>
+        public static int RunAsSudo(string commandLine)
```

Comment:
Talked offline with @JeremyKuhne and will let him decide about the point I am mentioning here, 

The written code here is specific for Linux as I am not expecting anyone calling sudo on Windows. so I suggest to abstract this AdminHelpers class and cross compile the implementation. to be specific it would be nice to call this file as AdminHelpers.Unix.cs and compile it for Linux only. and also create another file call it AdminHelpers.Windows.cs which will get compiled for Windows only. in the Windows file you can make RunAsSudo just throw exception. 

Also I suggest to rename RunAsSudo to be just Run to have make sense if we support  that on Windows later